app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,602 @@
+*= require_tree .
+
+/*general element styling*/
+ul, ol {
+  list-style: none;
+}
+
+html {
+  width: 100%;
+  height: 100%;
+}
+
+body {
+  width: 100%;
+  // background-color: black;
+  background: radial-gradient(ellipse at bottom, #1b2735 0%, #090a0f 100%);
+  color: white;
+  font-family: 'Josefin Slab', serif;
+}
+
+em {
+  font-weight: bold;
+  font-style: normal;
+}
+
+a, a:visited {
+  color: inherit;
+}
+
+a:hover {
+  color: grey;
+}
+
+/*flash message styling*/
+.flash, .status {
+  border-radius: 5px;
+  padding: 20px 25px;
+  background-color: transparent;
+  color: white;
+  font-size: 1.25rem;
+  font-family: 'Josefin Slab', serif;
+  text-align: center;
+}
+
+/*application wide header & dropdown menus*/
+header {
+  width: 100%;
+  margin-left: 0;
+  margin-right: 0;
+  padding-top: 1.5rem;
+  padding-bottom: 5rem;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+@media screen and (max-width: 480px) {
+  html {
+    background-color: black;
+  }
+
+  #stars, #stars2, #stars3 {
+    display: none
+  }
+
+  .logo {
+    display: none
+  }
+
+  .dropdown ul {
+    // display: none;
+  }
+}
+
+.logo a, .home-link {
+  /*font-family: 'Berkshire Swash', cursive;*/
+  font-family: 'Cinzel', serif;
+  color: white;
+  text-shadow: 10px;
+  font-weight: bold;
+  font-size: 4rem;
+  float: left;
+  transition: all .2s ease-in-out;
+}
+
+.dropdown {
+  position: relative;
+  display: flex;
+  text-align: left;
+  margin-left: 0;
+}
+
+.navigation .dropdown button {
+  font-family: 'Cinzel', serif;
+  font-size: 2em;
+  text-shadow: 1;
+  text-decoration: none;
+}
+
+.dropdown-button-browse, .dropdown-button-cart, .dropdown-button-login {
+  color: white;
+  padding: 0;
+  font-size: 3em;
+  border: none;
+  cursor: pointer;
+  font-weight: bolder;
+  margin: 0;
+  transition: all .2s ease-in-out;
+}
+
+.dropdown-button-browse:hover, .dropdown-button-cart:hover, .dropdown-button-login:hover, .logo a:hover {
+  transform: scale(1.15);
+}
+
+.dropdown-content-browse, .dropdown-content-cart, .dropdown-content-login {
+  display: none;
+  position: absolute;
+  font-family: 'Josefin Slab', serif;
+  color: white;
+  text-transform: capitalize;
+  font-size: 1.5rem;
+  width: 20em;
+  min-width: 160px;
+  text-align: left;
+  overflow: visible;
+  margin: 0;
+  z-index: 1;
+  transition: all .2s ease-in-out;
+}
+
+.dropdown-content-browse a, .dropdown-content-cart, .dropdown-content-login a {
+  padding: 12px 16px;
+  text-decoration: none;
+  width: 8em;
+  display: block;
+  transition: all .1s ease-in;
+}
+
+.dropdown-content-browse a:hover, .dropdown-content-merchant a:hover, .dropdown-content-cart, .dropdown-content-login a {
+  color: black;
+  width: 8em;
+  background-color: darkgrey;
+}
+
+.dropdown:hover .dropdown-content-browse {
+  display: block;
+}
+
+.dropdown:hover .dropdown-content-cart {
+  display: block;
+}
+
+.dropdown:hover .dropdown-content-cart {
+  display: block;
+}
```

Comment:
there are a couple of cases in this stylesheet that suffer from copy/paste and should be all on the same line, such as this:
```
.dropdown:hover .dropdown-content-browse, .dropdown:hover .dropdown-content-cart, .dropdown:hover .dropdown-content-cart {
  display: block;
}
```